src/main/java/teammates/common/datatransfer/questions/FeedbackRubricQuestionDetails.java

```diff
@@ -1181,6 +1247,7 @@ public String getCsvForSubQuestion(int subQuestion) {
             String subQuestionString = SanitizationHelper.sanitizeForCsv(alphabeticalIndex + ") "
                     + getRubricSubQuestions().get(subQuestion));
             DecimalFormat df = new DecimalFormat("0.00");
+            DecimalFormat dfWeight = new DecimalFormat("#.##");
```

Comment:
So the formatting of `weight` here may be different from the `html` format above (line 1207)?

Reply:
Sorry, I missed to add the same format for `html`.